northstar-tests/tests/tests.rs

```diff
@@ -413,39 +459,18 @@ test!(exitcodes, {
                 ..
             } if code == c)
         };
-        runtime.assume_notification(n, 5).await?;
-    }
-    runtime.shutdown().await
-});
-
-// Open many connections to the runtime
-test!(open_many_connections_to_the_runtime_and_shutdown, {
```

Comment:
noticed this test got removed, is that ok?

Reply:
The shutdown of the RT is no longer possible from within a test. Probably that's the reason why I removed this test. Hm. Think this test originally has been added because we had connections that stalled upon a RT shutdown.